pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -213,6 +213,18 @@ var mToF = []modelToFuncs{
 		displayFunc: ExecuteLedCmd,
 		arg:         "ACT,led0",
 	},
+	{
+		model:       "raspberrypi,4-compute-modulebrcm,bcm2711",
```

Comment:
Where do these names come from? I have tried to find the source of the name but cannot locate it anywhere. I see in runtime that it's taken from `/persist/status/hardwaremodel`, but how is this file populated?

Reply:
@OhmSpectator , these names come from `/proc/device-tree/compatible`, it's used by `GetHardwareModel()` along with other information retrieved from other sources (like SMBIOS), it's implemented here: https://github.com/lf-edge/eve/blob/master/pkg/pillar/hardware/model.go#L114 and https://github.com/lf-edge/eve/blob/master/pkg/pillar/hardware/model.go#L30, in this case the compatible string is reliable to identify the LED...

Reply:
Ah, I missed the code path... I thought we take it from the `hardwaremodel` file... 
And the ids themselves? I expected them to be available in some documentation/DTS files on the Internet but did not find anything.

Reply:
@OhmSpectator , it comes from the dts, like here: https://github.com/lf-edge/eve-kernel/blob/eve-kernel-arm64-v6.1.112-generic/arch/arm/boot/dts/bcm2711-rpi-cm4.dtsi#L8 but TBH I just got these values directly from the devices.... just did a `cat /proc/device-tree/compatible` :smile: